inputs.tf

```diff
@@ -12,6 +12,11 @@ variable "infrastructure_bucket_bastion_key" {
   default     = "bastion"
 }
 
+variable "infrastructure_bucket_github_users_script_key" {
```

Comment:
The `infrastructure_bucket_github_users_script_key` should probably be based inside of the `infrastructure_bucket_bastion_key` module input, as `infrastructure_bucket_bastion_key` reflects the Cuddlefish inventory. For example, `infrastructure_bucket_bastion_key` might be set to `default/bastion` or `working/bastion` - that's likely also where you want the Github users file to be stored.. Continuing my example, `infrastructure_bucket_github_users_script_key` may be set to `github-users`, instead of a complete S3 path to that file. IF you make this change, the description for that input should also note that this input is based in `infrastructure_bucket_bastion_key`.

Reply:
Good point. I simplified this so that the `infrastructure_bucket_github_users_script_key` variable doesn't actually exist anymore, but that instead the script is always placed inside of the `infrastructure_bucket_bastion_key` location and referenced as such.